addons/l10n_pt/data/account.group.template.csv

```diff
@@ -0,0 +1,146 @@
+id,code_prefix_start,code_prefix_end,name,chart_template_id/id
```

Comment:
no lines have a `code_prefix_end`, do we need it?

Reply:
Not needed as it is done here for instance https://github.com/odoo/odoo/blob/master/addons/l10n_be/data/account.group.template.csv :)

Reply:
But that example has a few `code_prefix_end`s. Does this file have any?

Reply:
Yes but only when the `code_prefix_end` is different from the `code_prefix_start` due to this:
https://github.com/odoo/odoo/blob/master/addons/account/models/account_account.py#L801-L804

Reply:
The `onchange` will not have any impact on an imported file. 
In cases where there are values in the field for at least one record, it makes sense to have it. But in this case, no records have a value in that field. 
There are many other l10n data files for account.group that have only the `_start` field, but not the `_end` - I am not sure why we need this field here if we're not providing any data.